contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -671,9 +682,12 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
     function _createActionInfo() internal view returns(ActionInfo memory) {
         ActionInfo memory rebalanceInfo;
 
-        // IMPORTANT: Compound oracle returns prices adjusted for decimals. USDC is 6 decimals so $1 * 10^18 * 10^18 / 10^6 = 10^30
-        rebalanceInfo.collateralPrice = strategy.priceOracle.getUnderlyingPrice(address(strategy.targetCollateralCToken));
-        rebalanceInfo.borrowPrice = strategy.priceOracle.getUnderlyingPrice(address(strategy.targetBorrowCToken));
+        // Calculate prices from chainlink. Adjusts decimals to be in line with Compound's oracles. Chainlink returns prices with 8 decimal places,
+        // but compound expects 36 + underlyingDecimals decimal places from their oracles. To adjust, we multiply by 36 - 8 - underlyingDeciamls
```

Comment:
Ok I think this makes sense, the one thing is that compound expects `36 - underlyingDecimals` for the _price_ so that when you multiply by the underlying _amount_ you end up with a total collateral valuation normalized to 36 decimals (ie the price for ETH should be 18 decimals, and for USDC to should be 30 decimals; but the USD valuation of the collateral is 36 decimals). 

Maybe make that distinction more clear